### ISSUE TITLE
Watch command fails due to incompatible wasm bindgen schema

### ISSUE THREAD

**Issue by adsick** [leptos-rs/cargo-leptos#129]
I tried updating everything many times but it won't help.

```cli
leptos/examples/counter_isomorphic on main [⇣] is 📦 v0.1.0 via 🦀 v1.71.0-nightly took 2m8s 
❯ cargo leptos watch
    Finished dev [unoptimized + debuginfo] target(s) in 0.10s
       Cargo finished cargo build --package=counter_isomorphic --lib --target-dir=target/front --target=wasm32-unknown-unknown --no-default-features --features=hydrate
Error: at `/home/adsick/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-leptos-0.1.8/src/compile/front.rs:42:30`

Caused by:
    0: at `/home/adsick/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-leptos-0.1.8/src/compile/front.rs:108:10`
    1: 
       
       it looks like the Rust project used to create this wasm file was linked against
       version of wasm-bindgen that uses a different bindgen format than this binary:
       
         rust wasm file schema version: 0.2.86
            this binary schema version: 0.2.84
       
       Currently the bindgen format is unstable enough that these two schema versions
       must exactly match. You can accomplish this by either updating the wasm-bindgen
       dependency or this binary.
       
       You should be able to update the wasm-bindgen dependency with:
       
           cargo update -p wasm-bindgen
       
       or you can update the binary with
       
           cargo install -f wasm-bindgen-cli
       
       if this warning fails to go away though and you're not sure what to do feel free
       to open an issue at https://github.com/rustwasm/wasm-bindgen/issues!
```

**Comment by adsick** [comment]
also a little note: that link at the end has an adjacent exclamation which gets inserted when you <kbd>Ctrl</kbd>+<kbd>click</kbd> on it in VS code leading to a 404 and confusion.

**Comment by jsprog** [comment]
Already faced this problem and got it resolved by:
- `cargo uninstall cargo-leptos`
- `cargo install --git https://github.com/leptos-rs/cargo-leptos --locked cargo-leptos`
- `cargo clean`

**Comment by adsick** [comment]
@jsprog actually only
```cargo install --git https://github.com/leptos-rs/cargo-leptos --locked cargo-leptos```
is enough to make it work, thanks. I believe we need to either mention that in the readme, or always make sure that cargo-leptos is always up to date on crates.io.

**Comment by jsprog** [comment]
@adsick

As stated in the logs you received, there was a conflict between:
- wasm-bindgen@0.2.84 as a dependency for cargo-leptos (tool)
- wasm-bindgen@0.2.86 as an indirect dependency on your project!

> `cargo install --git https://github.com/leptos-rs/cargo-leptos --locked cargo-leptos`
is enough to make it work, thanks

It should work due to the current dependency of the main branch (cargo-leptos) on wasm-bindgen@0.2.86

> I believe we need to either mention that in the readme

There is a hint about using the main branch from github if you've got **any reason**. But it's still an issue that kept me struggling for some time when testing some examples as a new comer to Leptos.

> If you, for **any reason**, need the bleeding-edge super fresh version:
>> `cargo install --git https://github.com/leptos-rs/cargo-leptos --locked cargo-leptos`

**Comment by akesson** [comment]
This should work with the newly released 0.1.9 (please re-open the issue otherwise)
